src/kayenta/report/detail/graph/semiotic/tooltip.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import * as ReactTooltip from 'react-tooltip';
+
+export interface ITooltipProps {
+  x?: number;
+  y?: number;
+  content?: any;
+}
+
+/*
+* Custom tooltip component to render on top of any svg component.
+* It creates an invisible pixel and uses react-tooltip library
+* to point to it.
+*/
+export default class Tooltip extends React.Component<ITooltipProps> {
+  private tooltipTarget: HTMLDivElement;
+
+  componentDidUpdate(prevProps: ITooltipProps) {
+    const target = this.tooltipTarget;
+    if (prevProps.content && !this.props.content) {
+      ReactTooltip.hide(target);
+    } else if (this.props.content) {
+      ReactTooltip.show(target);
+    }
+  }
+
+  render() {
+    const { x, y, content } = this.props;
+    const tooltipTargetStyle = {
+      left: x ? x : 0,
+      top: y ? y : 0,
+      height: 1,
+      width: 1,
+      opacity: 0,
+      pointerEvents: 'none',
+      position: 'absolute',
+      zIndex: 10,
+    } as React.CSSProperties;
+
+    const containerStyle = {
+      pointerEvents: 'none',
+    } as React.CSSProperties;
+
+    return (
+      <div style={containerStyle}>
+        <div data-tip={'tooltip'} style={tooltipTargetStyle} ref={el => (this.tooltipTarget = el)} />
```

Comment:
I just checked the documentation, and actually `data-tip` can simply be a boolean. There's also another optional attribute called 'data-for' on the target element that should match with `id` on the tooltip object. I'll add that as an optional prop in case we ever need to show multiple tooltips at once.